obspy/io/ascii/core.py

```diff
@@ -44,7 +44,7 @@
 from obspy.core.util import AttribDict, loadtxt
 
 
-HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %d sps, %.26s, %s, %s, %s\n"
+HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %s sps, %.26s, %s, %s, %s\n"
 
```

Comment:
any reason not to use `"%lg"` instead (mssed2ascii.c uses it for writing) - so the existing examples would be still correct